spec/components/schemas/reports/message/entry/base.ts

```diff
@@ -0,0 +1,43 @@
+// tslint:disable:max-line-length
+import { SchemaObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../../utils/ref';
+
+const base: SchemaObject = {
+  title: 'Entry object for Message Report',
+  description: 'This is a entry object model for Message Report.',
```

Comment:
Não lembro como tá nos outros objetos, mas o começo não tá tão legal. Acho que deveria ser:
"The entry object model for Message Report."

E acho que podia melhorar colocando:
"The entry object model for Message Report resource."